src/components/OptionList/README.md

```diff
@@ -175,14 +175,25 @@ function OptionListInPopoverExample() {
         onClose={togglePopoverActive}
       >
         <OptionList
+          role="listbox"
```

Comment:
Does this need some added documentation in this readme?

Reply:
Or is there a sane default? Just providing a string as the role seems something like we could improve on

Reply:
I am struggling with this because I think if it is an `OptionList` it should always be a `listbox` or `combobox`. However the role isn't used anywhere in `shopify/web`.

Reply:
What about we help out by keeping it optional but giving suggested types?

`type Role = 'listbox' | 'combobox' | string;`

```ts
interface Props {
  role?: Role;
}
```

Maybe we then remove the nullability in v6?